Adds registration and landmark annotation endpoints

HarmvZ opened the pull request:
- Adds an endpoint for getting all registrations(LandmarkAnnotations) for a list of images
- Adds an endpoint for getting the OCT/obs registrations for an OCT/obs image
See #768

jmsmkn commented on app/grandchallenge/retina_api/views.py:
```diff
@@ -776,3 +781,47 @@ def get_queryset(self):
+        queryset = user.landmarkannotationset_set.filter(
+            singlelandmarkannotation__image__id__in=image_ids
+        ).distinct()
+        # if len(queryset) == 0:
```
Please remove the commented out code

jmsmkn commented:
I think that you need to update the version of black that you're using. We've moved to 19.3b0 now. Also, I'd recommend installing the pre-commit hook which will do the formatting on your changesets automatically, would eliminate the `Run black` commits.

HarmvZ commented:
> I think that you need to update the version of black that you're using. We've moved to 19.3b0 now. Also, I'd recommend installing the pre-commit hook which will do the formatting on your changesets automatically, would eliminate the `Run black` commits.

I updated it already, but I think because this branch was made before the update it still runs on the [old version](https://github.com/comic/grand-challenge.org/blob/13d4f28122d0314a57e97319c2586565fb9d89fd/Pipfile#L12) in the `push` build.

I installed the pre-commit hook now, that will indeed make things a lot easier!

jmsmkn commented:
I see, thanks!
